docs/reference/compatibility/data-streams.md

```diff
@@ -0,0 +1,87 @@
+---
+navigation_title: Data streams comparison
+description: Learn how EDOT optimizes telemetry storage and query performance in Elastic Observability compared to classic APM.  
+applies_to:
+  stack:
+  serverless:
+    observability:
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-collector
+  - id: edot-sdk
+---
+
+# EDOT data streams compared to classic APM
+
+The Elastic Distribution of OpenTelemetry (EDOT) stores telemetry data using a storage model optimized for OpenTelemetry signals. When `mapping_mode: otel` is enabled, EDOT writes logs, traces, and metrics to specialized data streams aligned with OpenTelemetry semantics to improve efficiency.
+
+This architecture is designed for scalable observability workloads. It supports dynamic attributes, reduces mapping complexity, and avoids issues like mapping explosions or manual dimension setup.
+
+EDOT uses Elasticsearch’s [Logs data stream (LogsDB)](docs-content://manage-data/data-store/data-streams/logs-data-stream.md) and [Time Series Data Streams (TSDS)](docs-content://manage-data/data-store/data-streams/time-series-data-stream-tsds.md) as storage backends. These are purpose-built to handle the scale and variety of observability data.
+
+## Storage architecture
+
+EDOT uses specialized index modes for each type of telemetry signal:
+
+| Signal type | Index mode    | Storage backend |
+|-------------|---------------|------------------|
+| Logs        | `logsdb`      | LogsDB           |
+| Traces      | `logsdb`      | LogsDB           |
+| Metrics     | `time_series` | TSDS             |
+
+### Logs and traces in LogsDB
+
+Log and trace data is stored in LogsDB, a storage engine optimized for high-ingest, semi-structured observability data. Benefits include:
+
+* Improved indexing performance for dynamic fields like `attributes`  
+* Field-level sorting for more efficient queries  
+* Use of the [`passthrough`](#mapping-optimizations-with-passthrough) field type for flexible access without deep nesting  
+
+### Metrics in TSDS
+
+Metric data is stored using Elasticsearch’s TSDS. Benefits include:
+
+* Efficient storage using columnar compression  
+* Fast aggregations 
+* Automatic detection of metric dimensions (no need to manually define field sets)  
+
+## Mapping optimizations with `passthrough`
+
+OpenTelemetry attributes are stored using the [`passthrough`](elasticsearch://reference/elasticsearch/mapping-reference/passthrough.md) field type, enabling:
+
+* Full query support  
+* Dynamic mapping without immediate mapping explosions  
```

Comment:
The passthrough field type doesn't have an effect on whether or not there can be mapping explosions. The setting that helps against it is `ignore_dynamic_beyond_limit`. See also https://www.elastic.co/docs/manage-data/data-store/data-streams/logs-data-stream#logs-db-ignore-limit.